packages/webex/src/encryption.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) 2015-2024 Cisco Systems, Inc. See the LICENSE file.
+ */
+
+// Note: this file is written using commonjs instead of import/export to
+// simplify consumption by those less familiar with the current state of
+// JavaScript modularization
+
+/* istanbul ignore else */
+if (!global._babelPolyfill) {
+  /* eslint global-require: [0] */
+  require('@babel/polyfill');
+}
+
+require('@webex/plugin-authorization');
+// explicitly load wdm, since we're relying on preDiscoveryServices and the
+// url interceptor
+require('@webex/common');
+require('@webex/internal-plugin-encryption');
+require('@webex/plugin-encryption');
+require('node-scr');
```

Comment:
dont think we need node-src and internal encryption and common as they are already sub dependecny of the plugin encryption